test/js/node/net/node-net-server.test.ts

```diff
@@ -187,20 +187,20 @@ describe("net.createServer listen", () => {
     const server: Server = createServer();
 
     let timeout: Timer;
-    const closeAndFail = () => {
+    const closeAndFail = (err: any) => {
       clearTimeout(timeout);
       server.close();
-      mustNotCall()();
+      mustNotCall(err ? err.toString() : "timeout")();
     };
     server.on("error", closeAndFail);
     timeout = setTimeout(closeAndFail, 100);
 
     server.listen(
-      49027,
+      49028,
```

Comment:
why the hardcoded port number? this test will be flaky

Reply:
I think the ideia was testing if was working on some specific port, but we can remove this tests with fixed port is just flasky

Reply:
removed here https://github.com/oven-sh/bun/pull/3457/commits/7c49b2676260d1156896be0d326e2bef9222cbf1